region/maridia/outer.json

```diff
@@ -1297,6 +1356,30 @@
         {
           "from": 9,
           "to": [
+            {
+              "id": 1,
+              "requires": [
+                {"canComeInCharged":{
+                  "fromNode": 5,
+                  "framesRemaining": 90,
+                  "shinesparkFrames": 65
+                }}
```

Comment:
These two sparks probably require `canSwim` or `canSuitlessMaridia` like many of the other ones?

Reply:
Yus